src/app/Caches/Create/Features/IndexedCache.tsx

```diff
@@ -0,0 +1,396 @@
+
+    const [isOpenIndexReader, setIsOpenIndexReader] = useState(false);
+    const [isOpenIndexWriter, setIsOpenIndexWriter] = useState(false);
+    const [isOpenIndexMerge, setIsOpenIndexMerge] = useState(false);
+    const [entityInput, setEntityInput] = useState<string>('');
+
+    useEffect(() => {
+        props.indexedOptionsModifier({
+            enableIndexing: enableIndexing,
+            indexedStorage: indexedStorage,
+            indexedEntities: indexedEntities,
+            indexReader: refreshInterval,
+            indexWriter: {
+                commitInterval: commitInterval,
+                lowLevelTrace: lowLevelTrace,
+                maxBufferedEntries: maxBufferedEntries,
+                queueCount: queueCount,
+                queueSize: queueSize,
+                ramBufferSize: ramBufferSize,
+                threadPoolSize: threadPoolSize,
+            },
+            indexMerge: {
+                calibrateByDeletes: calibrateByDeletes,
+                factor: factor,
+                maxEntries: maxEntries,
+                minSize: minSize,
+                maxSize: maxSize,
+                maxForcedSize: maxForcedSize,
+            },
+        });
+    }, [enableIndexing, indexedStorage, indexedEntities, refreshInterval, commitInterval, lowLevelTrace, maxBufferedEntries, queueCount, queueSize, ramBufferSize, threadPoolSize, calibrateByDeletes, factor, maxEntries, minSize, maxSize, maxForcedSize]);
+
+    const deleteChip = (chipToDelete: string) => {
+        const newChips = indexedEntities.filter(chip => !Object.is(chip, chipToDelete));
+        setIndexedEntities(newChips);
+    };
+
+    const clearChipsAndInput = () => {
+        setIndexedEntities([]);
+        setEntityInput('');
+    };
+
+    const addChip = (newChipText: string) => {
+        setIndexedEntities([...indexedEntities, `${newChipText}`]);
+        setEntityInput('');
+    };
+
+    /** enable keyboard only usage while focused on the text input */
+    const handleTextInputKeyDown = (event: React.KeyboardEvent) => {
+        if (event.key === 'Enter' && entityInput.length) {
+            addChip(entityInput);
+        }
+    };
+
+    const showClearButton = !!entityInput || !!indexedEntities.length;
+
+    /** render the utilities component only when a component it contains is being rendered */
+    const showUtilities = showClearButton;
+
+    const formIndexEntities = () => {
+        return (
+            <FormGroup fieldId='indexed-entities'>
+                <MoreInfoTooltip label="Indexed Entities" toolTip="Indexed Entities Tooltip" textComponent={TextVariants.h2} />
+                <TextInputGroup>
+                    <TextInputGroupMain
+                        value={entityInput}
+                        onChange={(val) => setEntityInput(val)}
+                        onKeyDown={handleTextInputKeyDown}
+                        placeholder="search"
+                        aria-label="Search input"
```

Comment:
As I explained, can't be a search. You need to provide a textfield where the user will type de value (for example com.redhat.PurchaseOrderItem), click on a button "add" and display them as label or chip so it can be removed or changed. I think label might be better as they are editable now too (beta feature) https://www.patternfly.org/v4/components/label

Reply:
To add item, user can enter after entering in text field, should we also need "add" button?